tests/test_model_class.py

```diff
@@ -54,9 +54,11 @@ def _compare_wflow_models(mod0, mod1):
                     else f"{map1.dtype} instead of {map0.dtype}"
                 )
                 # Check on nodata
+                # hilariously np.nan == np.nan returns False, hence the additional check
```

Comment:
funny indeed ^^ we barely use nan as a nodata value so guess that's why so far we did not see this